recipe/meta.yaml

```
@@ -42,6 +42,7 @@ requirements:
     - mpifx * mpi_{{ mpi }}_*  # [mpi != "nompi"]
     - scalapack  # [mpi != "nompi"]
     - scalapackfx * mpi_{{ mpi }}_*  # [mpi != "nompi"]
+    - libnegf * {{ mpi_prefix }}_*
```

The correct `libnegf` version is picked automatically for non-MPI builds, constraints only apply for MPI builds:
```suggestion
    - libnegf
    - libnegf * {{ mpi_prefix }}_*  # [mpi != "nompi"]
```